### PR TITLE
Update authentication proxy requirements

### PR DESCRIPTION
**Pull request by liggitt**
Docs for https://github.com/openshift/origin/pull/13569

We need this in the 1.5 and 1.4 docs as well

### CODE REVIEW

File: install_config/configuring_authentication.adoc [hunk 1]
```diff
@@ -722,13 +723,15 @@ respond with a login flow.
...
+that will authenticate browser-based clients and then proxy their request to _https://<master>/oauth/authorize_.
+The URL that proxies to _https://<master>/oauth/authorize_ must end with _/authorize_ (with no trailing slash),
+and also proxy subpaths, in order for OAuth approval flows to work properly.
+_${url}_ is replaced with the current URL, escaped to be safe in a query parameter.
+_${query}_ is replaced with the current query string.
 <6> Optional: URL to redirect unauthenticated _/oauth/authorize_ requests to,
-for clients which expect `WWW-Authenticate` challenges. _${url}_ is replaced
-with the current URL, escaped to be safe in a query parameter. _${query}_ is
-replaced with the current query string.
+that will clients which expect `WWW-Authenticate` challenges, and then proxy them to _https://<master>/oauth/authorize_.
```
Comment (adellape) [inline install_config/configuring_authentication.adoc]: Can you check this sentence? "that will clients which expect"

**Comment by adellape** [comment]
@liggitt One question inline, but I'll take this PR over to do some edits to the example URL formatting.

File: install_config/configuring_authentication.adoc (hunk 1 above)
Comment (liggitt) [inline install_config/configuring_authentication.adoc]: fixed

**Comment by adellape** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/4086.

**Pull request closed without merging**